src/AutoRest.CSharp/Output/Models/Types/ObjectType.cs

```diff
@@ -86,6 +86,8 @@ public ObjectType(ObjectSchema objectSchema, BuildContext context): base(context
 
         public ObjectTypeProperty[] Properties => _properties ??= BuildProperties().ToArray();
 
+        public SchemaTypeUsage SchemaTypeUsage => _usage;
```

Comment:
I wonder if we should put this value onto the `JsonSerialization` inside the `        public ObjectSerialization[] Serializations => _serializations ??= BuildSerializations();
`

Reply:
Or better follow the pattern of IncludeSerializer method https://github.com/Azure/autorest.csharp/blob/4c0f9cf31af56decb03c67fdf99eb516c8ae81ae/src/AutoRest.CSharp/Output/Models/Types/ObjectType.cs#L306-L307